internal/codespaces/codespaces.go

```diff
@@ -90,7 +90,7 @@ func ConnectToLiveshare(ctx context.Context, progress progressIndicator, session
 
 // ListenTCP starts a localhost tcp listener and returns the listener and bound port
 func ListenTCP(port int) (*net.TCPListener, int, error) {
-	addr, err := net.ResolveTCPAddr("tcp", fmt.Sprintf("127.0.0.1:%d", port))
+	addr, err := net.ResolveTCPAddr("tcp", fmt.Sprintf(":%d", port))
```

Comment:
Would we consider scoping this change to just `ForwardPorts` and keep all other call points passing `127.0.0.1`? That's how it was before https://github.com/cli/cli/pull/6888, so maybe it would avoid unforeseen regressions

Perhaps ListenTCP could accept e.g. an `allInterfaces bool` parameter that allows binding a port on all network interfaces instead of just to the `127.0.0.1` one.

Reply:
Had considered that too - I did manually test the other commands and they looked good, but I'm OK to duplicate some code to avoid regression-whack-a-mole 😄

Reply:
Pulled out a PF specific version